HousingRepairsOnlineApi/SoRConfig.json

```diff
@@ -90,8 +115,16 @@
       "lights":"N858111",
       "sockets": "N861511"
     },
-    "dampOrMould": {
-      "dampMouldCausedByOther":"N114001"
+    "dampOrMould": "N114001",
+    "damagedOrStuckWindows": {
+      "smashed": "",
```

Comment:
Should this instance of `smashed` and `stuckOpen` have the same SoRs as other instances? i.e. `N551055` and `N315001` respectively.